### DISCUSSION TITLE
Installation wiki doesn't mention SSHFS

### DISCUSSION THREAD

**Discussion by Rillian-Grant** [ltsp/ltsp#167] [Q&A]

**Comment by alkisg** [comment]
Hi, sshfs is listed as a dependency of the ltsp package:
https://github.com/ltsp/ltsp/blob/master/debian/control#L20

...so it's always automatically installed along with ltsp.

**Comment by Rillian-Grant** [comment]
But it does not mention how you would use SSHFS instead of NFS

**Comment by alkisg** [comment]
I'm not sure what you mean and what page you are referring to.
https://github.com/ltsp/ltsp/wiki/installation suggests the default setup, which is NFS for /srv/ltsp, and SSHFS for /home.
Both of those dependencies properly get installed by following the installation page instructions.

What/where exactly is the problem?
